### PR TITLE
French translation for the `forced-color-adjust` CSS property reference

### PR DESCRIPTION
**Pull request by audrasjb**
This PR creates a French translation for the `forced-color-adjust` CSS property reference.

### CODE REVIEW

**Comment by audrasjb** [comment]
Just added another commit to fix the glossary issue (see detected Flaws)

**Review by SphinxKnight — CHANGES_REQUESTED** [review]
The embedded image should be resolved from the English repo, I believe it could be deleted here (as it's the same as en-US).

I'm aware of few edge cases (e.g. when images are embedded in some macros/examples) here so don't hesitate to deny this :x (I hope \<figure> is not one of them)

File: files/fr/web/css/forced-color-adjust/index.html [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+<dl>
+	<dt><code>auto</code></dt>
+	<dd>En mode de couleurs forcées, les couleurs de l'élément sont définies par l'{{Glossary("user agent", "agent utilisateur")}}. C'est la valeur par défaut.</dd>
+	<dt><code><strong>none</strong></code></dt>
+	<dd>Même en mode de couleurs forcées, les couleurs de l'élément ne sont pas automatiquement définies par l'{{Glossary("user agent", "agent utilisateur")}}.</dd>
+</dl>
+
+<h2 id="usage_notes">Notes d'utilisation</h2>
+
+<p>Cette propriété ne devrait être utilisée que pour faire des modifications qui prendront en charge les pré-requis de couleurs et de contrastes. Par exemple, si vous savez que les optimisations de couleur faites par l'{{Glossary("user agent", "agent utilisateur")}} conduisent à une mauvaise expérience en mode contrasté ou en mode sombre. L'utilisation de cette propriété vous permettra alors de modifier le résultat dans ce mode pour proposer une meilleure expérience. <strong>Elle ne devrait pas être utilisée dans le but d'empêcher les choix de l'utilisateur d'être respectés.</strong>.</p>
```
Comment (SphinxKnight) [inline files/fr/web/css/forced-color-adjust/index.html:35]: superfluous dot at the end of the paragraph

**Comment by audrasjb** [comment]
> The embedded image should be resolved from the English repo, I believe it could be deleted here (as it's the same as en-US).
> 
> I'm aware of few edge cases (e.g. when images are embedded in some macros/examples) here so don't hesitate to deny this :x (I hope <figure> is not one of them)

Yes, it works without the image!
_still learnin'_ :D

File: files/fr/web/css/forced-color-adjust/index.html (hunk 1 above)
Comment (audrasjb) [reply files/fr/web/css/forced-color-adjust/index.html]: Good point! Fixed in https://github.com/mdn/translated-content/pull/1745/commits/8c4d18619131174f5e17ffa4064c2448676eea75

**Review by SphinxKnight — APPROVED** [review]
🙇‍♂️

**Pull request merged**